code/game/area/Space Station 13 areas.dm

```diff
@@ -977,8 +977,8 @@ proc/process_ghost_teleport_locs()
 	icon_state = "server"
 
 /area/toxins/telesci

```

Comment:
Well it technically is still telesci... Lol, no I just forgot to rename that part.